src/plugin.cc

```diff
@@ -75,6 +77,11 @@ class OurGlueCompiler : public spicy::zeek::GlueCompiler {
 };
 
 plugin::Zeek_Spicy::Plugin::Plugin() {
+    // This ensures version compatibility at dlopen() time  by requiring the
+    // versioned symbol to be present.
+    _spicy_version = HILTI_VERSION_FUNCTION();
+
+    // Not sure if with the with the check above, we still need this? Can't hurt I guess.
```

Comment:
This seems to check another compatibility aspect, so this should either already have been handled by the Zeek version functionality or we should instead evaluate that symbol like we do with `HILTI_VERSION_FUNCTION`.

Reply:
Not sure what you mean by "evaluate that symbol like we do with HILTI_VERSION_FUNCTION"?

Reply:
I am unsure whether the symbol is evaluated automatically with the Zeek-side tooling we are using, or whether we need code like `_spicy_version = HILTI_VERSION_FUNCTION();` but for the Zeek version function.

Reply:
Ah, Zeek's plugin integration takes care of calling the Zeek version function.